test/test_memory.cpp

```diff
@@ -25,18 +25,21 @@
 
 namespace nvfuser {
 
-class MemoryTest : public NVFuserTest {};
+class MemoryTest
+    : public testing::TestWithParam<std::tuple<CacheOp, std::string>> {};
```

Comment:
This seems useful. Can we create something like `NvFuserTestWithParam` too? There are a couple of things we want to do in `SetUp` and `TearDown`.

Reply:
Done.